### ISSUE TITLE
Copy & paste crashes the app

### ISSUE THREAD

**Issue by FlorianLudwig** [elevenhsoft/WebApps#31]
When pasting into the app, the app crashes.

system:
- gnome
- wayland
- App installed via flatpak

**Comment by elevenhsoft** [comment]
Yes, it's known issue on libcosmic. Probably we must wait upstream since I'm not sure if I'm able to fix it :)
